docker/base/launchTomcat.sh

```diff
@@ -67,31 +74,35 @@ if [ "$1" = "--help" ] || [ "$1" = "-help" ] || [ "$1" = "-h" ]; then
     exit 0
 fi
 
-if [ "x$FIREFLY_SHARED_WORK_DIR" != "x" ]; then
-      mkdir -p ${FIREFLY_SHARED_WORK_DIR}
-fi
-
-
 if [ -z ${MAX_JVM_SIZE} ]; then
    JVM_SIZING="-XX:InitialRAMPercentage=${INIT_RAM_PERCENT} -XX:MaxRAMPercentage=${MAX_RAM_PERCENT}"
 else
    JVM_SIZING="-Xms${MIN_JVM_SIZE} -Xmx${MAX_JVM_SIZE}"
 fi
 
+if [ -z ${VISUALIZE_FITS_SEARCH_PATH} ]; then
+   VIS_PATH="/external"
+else
+   VIS_PATH="/external:${VISUALIZE_FITS_SEARCH_PATH}"
+fi
+
 #---------   CATALINA_OPTS must be exported for catalina.sh to pick them up
 export CATALINA_OPTS="\
-        ${JVM_SIZING} \
-        -Dserver.cores=${JVM_CORES} \
-        -Djava.net.preferIPv4Stack=true \
-        -Dnet.sf.ehcache.enableShutdownHook=true \
-        -Dserver_config_dir=${SERVER_CONFIG_DIR} \
-        -Dwork.directory=${FIREFLY_WORK_DIR} \
-        -Dshared.work.directory=${FIREFLY_SHARED_WORK_DIR} \
-        -Dvisualize.fits.search.path=${VISUALIZE_FITS_SEARCH_PATH} \
-	   ${FIREFLY_OPTS}"
+  ${JVM_SIZING} \
+  -Dhost.name=${HOSTNAME} \
+  -Dserver.cores=${JVM_CORES} \
+  -Djava.net.preferIPv4Stack=true \
+  -Dwork.directory=/firefly/workarea \
+  -Dshared.work.directory=/firefly/shared-workarea \
+  -Dserver_config_dir=/firefly/config \
+  -Dstats.log.dir=/firefly/logs/statistics \
+  -Dalerts.dir=/firefly/alerts \
+  -Dvisualize.fits.search.path=${VIS_PATH} \
```

Comment:
since you made this change in statistics, does this mean that we will not attempt to make a statistics dir if the use just maps logs?

Reply:
If they mount /firefly/logs, then we would create a statistics directory under logs if one does not exists. Permission is not a problem since we expect the process owner to have write access to this directory.
If they want only the stats logs to persistent outside the container, then they can mount /firefly/statistics instead.